cibuildwheel/linux.py

```diff
@@ -79,14 +81,14 @@ def build(project_dir, output_dir, test_command, test_requires, before_build, bu
                 # the first element
                 if [[ "$built_wheel" == *none-any.whl ]]; then
                     # pure python wheel - just copy
-                    mv "$built_wheel" /tmp/delocated_wheel
+                    mv "$built_wheel" /tmp/delocated_wheels
                 else
-                    auditwheel repair "$built_wheel" -w /tmp/delocated_wheel
+                    auditwheel repair "$built_wheel" -w /tmp/delocated_wheels
                 fi
-                delocated_wheel=(/tmp/delocated_wheel/*.whl)
+                delocated_wheels=(/tmp/delocated_wheels/*.whl)
 
                 # Install the wheel we just built
-                "$PYBIN/pip" install "$delocated_wheel"
+                "$PYBIN/pip" install "${{delocated_wheels[0]}}"
```

Comment:
arg, this is tricksy... do we need to predefine a precedence here? Although, if auditwheel produces two wheels, they're the same. If that's true, we just need to comment it here, something like `# if auditwheel produced two wheels, it's because its output conforms to multiple manylinux standards. They're functionally the same, so it doesn't matter which one we run the tests on.`

Reply:
That's what I was thinking indeed.
They should be the same. Only the `pip` detection/installation is influenced by this, I guess. But we're testing on the same platform as we're building, so in this regard, it's not a good test anyway.